dlt-connector/src/data/Transaction.logic.test.ts

```diff
@@ -0,0 +1,323 @@
+import { Community } from '@entity/Community'
+import { Transaction } from '@entity/Transaction'
+import { Decimal } from 'decimal.js-light'
+
+import { logger } from '@/logging/logger'
+
+import { CommunityRoot } from './proto/3_3/CommunityRoot'
+import { CrossGroupType } from './proto/3_3/enum/CrossGroupType'
+import { GradidoCreation } from './proto/3_3/GradidoCreation'
+import { GradidoDeferredTransfer } from './proto/3_3/GradidoDeferredTransfer'
+import { GradidoTransfer } from './proto/3_3/GradidoTransfer'
+import { RegisterAddress } from './proto/3_3/RegisterAddress'
+import { TransactionBody } from './proto/3_3/TransactionBody'
+import { TransactionLogic } from './Transaction.logic'
+
+let a: Transaction
+let b: Transaction
+
+describe('data/transaction.logic', () => {
+  describe('isBelongTogether', () => {
+    beforeEach(() => {
+      const now = new Date()
+      let body = new TransactionBody()
+      body.type = CrossGroupType.OUTBOUND
+      body.transfer = new GradidoTransfer()
+      body.otherGroup = 'recipient group'
+
+      a = new Transaction()
+      a.community = new Community()
+      a.communityId = 1
+      a.otherCommunityId = 2
+      a.id = 1
+      a.signingAccountId = 1
+      a.recipientAccountId = 2
+      a.createdAt = now
+      a.amount = new Decimal('100')
+      a.signature = Buffer.alloc(64)
+      a.bodyBytes = Buffer.from(TransactionBody.encode(body).finish())
+
+      body = new TransactionBody()
+      body.type = CrossGroupType.INBOUND
+      body.transfer = new GradidoTransfer()
+      body.otherGroup = 'sending group'
+
+      b = new Transaction()
+      b.community = new Community()
+      b.communityId = 1
+      b.otherCommunityId = 2
```

Comment:
Oh you right, the community and otherCommunity fields not clearly named.
community contain the community of signer, and otherCommunity of recipient.
I've updated my code so that community is the community on which blockchain the transaction will be send and otherCommunity get's the same community as in TransactionBody.otherGroup,
the community in which the linked transaction can be found.

Why with account ids?
They are correctly named signingAccountId and recipientAccountId.
The signer is always the same, regardless if we on the INBOUND or OUTBOUND Transaction. The same for the recipient.